<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinearRegression</class>
 <widget class="QMainWindow" name="LinearRegression">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>255</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>285</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>285</width>
    <height>255</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Regresja liniowa</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="windowLabel">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Regresja liniowa</string>
    </property>
   </widget>
   <widget class="QLabel" name="coordinatesLabel">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>35</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Współrzędne punktów (x y; x y; ...):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="coordinatesEditBox">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>55</y>
      <width>256</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0 0</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="methodGroupBox">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>80</y>
      <width>256</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Metoda regresji</string>
    </property>
    <widget class="QRadioButton" name="secondTypeRadioButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>drugiego rodzaju</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="orthogonalRadioButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ortogonalna</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="computeButton">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>130</y>
      <width>256</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Oblicz</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>165</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wynik:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="resultTextBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>185</y>
      <width>256</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>285</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <widget class="QMenu" name="menuPrzyklad">
     <property name="title">
      <string>Przyklad</string>
     </property>
     <addaction name="actionSecond_Moment"/>
     <addaction name="actionOrthogonal"/>
    </widget>
    <addaction name="actionInstrukcja"/>
    <addaction name="menuPrzyklad"/>
   </widget>
   <addaction name="menuPomoc"/>
  </widget>
  <action name="actionInstrukcja">
   <property name="text">
    <string>Instrukcja</string>
   </property>
  </action>
  <action name="actionSecond_Moment">
   <property name="text">
    <string>Drugiego rodzaju</string>
   </property>
  </action>
  <action name="actionOrthogonal">
   <property name="text">
    <string>Ortogonalna</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
